exporter/metric/metric_test.go

```diff
@@ -39,10 +41,23 @@ var (
 )
 
 func TestDescToMetricType(t *testing.T) {
+	cloudMock := cloudmock.NewCloudMock()
```

Comment:
This test only invokes the `descToMetricType()` function, i.e. it doesn't export anything, so setting up a mock seems unnecessary (i.e. you can just revert changes in this test).